sale_product_company/migrations/16.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Tecnativa - Pilar Vargas
```

Comment:
You should bump manually the version on the manifest, and put such version here in the folder name.